addons/test_mail/tests/test_mail_followers.py

```diff
@@ -56,6 +56,8 @@ def test_field_message_partner_ids(self):
         followed_after = self.env['mail.test.simple'].search([('message_partner_ids', 'in', partner.ids)])
         self.assertTrue(partner in test_record.message_partner_ids)
         self.assertEqual(followed_before + test_record, followed_after)
+        message_partner = self.env['mail.thread']._search_message_partner_ids('in', [])
```

Comment:
Probably better to use the classic search method. ``_search_message_partner_ids`` is the technical implementation code, but what matters is that search works as intended, isn't it ?

Reply:
Hello @tde-banana-odoo, yeah we can have the search instead of the technical implementation, I've just wanted to make what I'm testing more explicit.
My goal is to secure the _search method because, prior to version 16, it would sometimes return a list or an integer instead of a query object. Therefore, this pull request is intended only for version 15. after that (starting from 16), the _search now always returns a query object.
If you still prefer to use the classic search method, I can still do it, I just wanted the test to be clear.
and thank you for your review :)